event-log-dispatcher/src/main/scala/com/ubirch/dispatcher/process/DispatchExecutor.scala

```diff
@@ -118,18 +119,20 @@ class DispatchExecutor @Inject() (
         throw EmptyValueException("No Records Found to be processed", pipeData)
       }
 
-      v1.foreach(x => run(x).runOnComplete {
```

Comment:
`runOnComplete` is duplicated

Reply:
where is it duplicated?

Reply:
![duplicated_runOnComplete](https://github.com/ubirch/ubirch-event-log/assets/11785972/b7b2f0fe-ce3d-42b8-865e-a9367367e1c2)